src/components/Organisms/TodoCard/index.jsx

```diff
@@ -33,6 +36,16 @@ export const TodoCard = () => {
     setTaskList(newTaskList);
   };
 
+  useEffect(() => {
+    const latest = JSON.parse(localStorage.getItem("storage"));
+    if (latest === null) return;
+    setTaskList(latest);
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem("storage", JSON.stringify(taskList));
+  }, [taskList]);
```

Comment:
```suggestion
  useEffect(() => {
    const storedTaskList = JSON.parse(localStorage.getItem("taskList"));
    setTaskList(storedTaskList ?? []);
  }, []);

  useEffect(() => {
    localStorage.setItem("taskList", JSON.stringify(taskList));
  }, [taskList]);
```
ストレージの名前がちょっと汎用的すぎる感じがしたので、
自分だったらこのように書くかな〜というのを記載させていただいています。

Null 合体演算子を使うことでコードをシンプルにできるので、このように色々な書き方ができるようになっておくのがおすすめです！
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing